packages/account/src/Components/poi/poi-form-on-signup/idv-doc-submit-on-signup/idv-doc-submit-on-signup.tsx

```diff
@@ -141,6 +143,8 @@ export const IdvDocSubmitOnSignup = ({
                                 is_appstore
                                 should_hide_helper_image={shouldHideHelperImage(values?.document_type?.id)}
                                 editable_fields={changeable_fields}
+                                residence_list={residence_list}
+                                warning_items={undefined}
```

Comment:
I think its better to set the default value of warning_items as empty object rather than undefined?